### DISCUSSION TITLE
Github Android App doesn't login

### DISCUSSION THREAD

**Discussion by msoareseq** [community/community#122856] [Mobile]
### Select Topic Area

Bug

### Body

I try to login at Github mobile app. The login page opens in Chrome. I log it as usual using the Microsoft Authenticator generated code. So far so good. Chrome reopens the app, and it keeps loading for several seconds, and then gives me a error, just saying "There was an error login in Github. Please try again. I tried like 30 times and keeps saying the same thing. Uninstalled the app and installed it again, and keeps sohwing the same error. Cleared Chrome cache, doesn't work. At desktop I can log in normally. This app seems useless now.

**Comment by 22coffeeboi** [comment]
It seems like you're encountering a persistent login issue with the GitHub mobile app on your device, where despite repeated attempts, you're unable to log in successfully. Despite reinstalling the app and clearing the Chrome cache, the problem persists. Since you're able to log in normally via desktop, it appears to be specific to the mobile app.

This issue could potentially be caused by various factors, such as network connectivity issues, device-specific configurations, or bugs within the app itself. To address this problem effectively, you might consider the following steps:

Check Network Connectivity: Ensure that your mobile device has a stable internet connection, as network disruptions can sometimes interfere with the login process.
Update the App: Make sure that the GitHub app is updated to the latest version available on your device's app store. Developers often release updates to fix bugs and improve app performance.
Contact GitHub Support: If the issue persists after trying the above steps, consider reaching out to GitHub's support team for assistance. They may be able to provide further insights or solutions tailored to your specific situation.
Alternative Login Methods: As a temporary workaround, you could try logging in using alternative methods, such as email and password authentication, to see if the issue is specific to the Microsoft Authenticator-generated code.
By following these steps and potentially seeking assistance from GitHub support, you may be able to resolve the login issue and regain access to the mobile app's functionality

**Comment by HenSquared** [comment]
Hi @msoareseq, 

I've opened an issue with the Android team about this. 

Has this issue started recently for you?

**Comment by msoareseq** [comment]
Hi Henry!Thank you very much for your help!I noticed it yesterday, when I tried to login on a new PC using GitHub Mobile Code. The app didn't show the notification. So I open it, and it kept a endless loading my profile. Só I decided to do a logout, and when I tried to login again, the problem started...I uninstalled it, and installed again, but no success. Em 8 de mai. de 2024 11:51, Henry ***@***.***> escreveu:
Hi @msoareseq,
I've opened an issue with the Android team about this.
Has this issue started recently for you?

**Reply by alcere** [reply]
Hi @msoareseq, can you share with us your device model and the GirHUb app version you are running, please? Thanks!

**Comment by msoareseq** [comment]
Hi Alfredo!

Thank you very much for the attention.
The issue occurs on an ASUS Zenfone 9 3.2GHz 8GB RAM, 256GB Storage – Android 14
I can’t confirm directly the version of the app I am using (because it don’t login). But I saw at Google Play Store and the last update for Github Mobile was April 30th 2024.

**Reply by alcere** [reply]
@msoareseq is this still happening?

**Reply by msoareseq** [reply]
No. It's working now. Never had this problem again.

**Reply by alcere** [reply]
Glad to hear that! Please keep the feedback coming!
